core-common/src/main/java/org/apache/kylin/common/util/DateFormat.java

```diff
@@ -194,4 +202,18 @@ public static boolean isDatePattern(String ptn) {
         return COMPACT_DATE_PATTERN.equals(ptn) || YYYYMMDDHH.equals(ptn) || YYYYMMDDHHMM.equals(ptn)
                 || YYYYMMDDHHMMSS.equals(ptn);
     }
+
+    public static Long getFormatTimeStamp(long time, String pattern) {
+        try {
+            if (StringUtils.isNotBlank(pattern)) {
+                SimpleDateFormat sdf = new SimpleDateFormat(pattern, Locale.getDefault(Locale.Category.FORMAT));
+                sdf.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
This is hard code for timezone, please reference to org.apache.kylin.common.KylinConfigBase#getTimeZone

Reply:
![b834587586ad4ff0235117f3887cbb5](https://user-images.githubusercontent.com/49258176/209627680-9689cecb-9e23-4e96-bb41-14c8053dbd34.png)